src/apps/bridge/bridgePowerController.h

```diff
@@ -45,16 +45,15 @@ class BridgePowerController {
   static constexpr uint32_t DEFAULT_SUBSAMPLE_ENABLED = 0;
   static constexpr uint32_t DEFAULT_SUBSAMPLE_INTERVAL_S = (60);
   static constexpr uint32_t DEFAULT_SUBSAMPLE_DURATION_S = (30);
-  static constexpr uint32_t MIN_SAMPLE_INTERVAL_S = (5 * 60);
-  static constexpr uint32_t MIN_SAMPLE_DURATION_S = (60);
+  static constexpr uint32_t MIN_SAMPLE_DURATION_S = (1);
+  static constexpr uint32_t MIN_SAMPLE_INTERVAL_S = (1);
   static constexpr uint32_t MAX_SAMPLE_INTERVAL_S = (24 * 60 * 60);
   static constexpr uint32_t MAX_SAMPLE_DURATION_S = (24 * 60 * 60);
-  static constexpr uint32_t MIN_SUBSAMPLE_INTERVAL_S = (12);
-  static constexpr uint32_t MIN_SUBSAMPLE_DURATION_S = (6);
+  static constexpr uint32_t MIN_SUBSAMPLE_INTERVAL_S = (1);
+  static constexpr uint32_t MIN_SUBSAMPLE_DURATION_S = (1);
```

Comment:
I believe I made the min sample/subsample durations 6s to make sure whenever the bus is turned on we have sufficient time do a topology sample. Have you tested that we are still able to get the topology with it down at 1?

Reply:
Sounds good!

Reply:
Our status quo for configs: we've been recommending settings, but giving folks flexibility to play with extremes. I don't expect people to set to the minimum, but if they do -- it is a devkit after all